.github/workflows/test.yml

```diff
@@ -13,7 +13,7 @@ jobs:
     name: Test
     strategy:
       matrix:
-        os: [ubuntu-latest, windows-latest]
+        os: [ubuntu-latest]
```

Comment:
verify once if we rely on paths or any OS specific code base lets remove if gtg

Reply:
verified